### DISCUSSION TITLE
function edited to show download invoice PDF with if condition

### DISCUSSION THREAD

**Discussion by faridfox** [wpovernight/woocommerce-pdf-invoices-packing-slips#193] [Show and tell]
I made a lil gist here:

https://gist.github.com/faridfox/de08edaf4c5e7acb4933ed7bc4c7973f

to show Download invoice PDF with if condition.
Thank you @wpovernight

**Comment by Spreeuw** [comment]
thanks for sharing that @faridfox !
Two notes:

1. If you include `<?php` above the code, gist will automatically format it as PHP code
2. The code in your `else` statement _only_ works when you have enabled Guest access in the Status tab of the plugin settings. And when you have already enabled that, you don't actually need the first part of the `if` statement, because the second part works for both guests and logged in users.

**Reply by faridfox** [reply]
Thanks for note about gist :) I'm just new here
about if condition I did it because I thought if I didn't mention guest condition Download link will not appear.
but now you made it clear. I will give it a try.

Do you suggest removing the gis?
P.S. I suggest that you mention "the second part works for both guests and logged in users." in your documentation or elsewhere.

Thank you for the awesome plugin. Keep coding

**Reply by Spreeuw** [reply]
Yes, I would recommend just using the second snippet since it works for both. I have amended the documentation with your suggestion, thanks!
